documentation/docs/Contributing/Contributing_to_repository/development-pipeline.md

```diff
@@ -72,10 +72,10 @@ Add relevant personnel to assignees section, assignees will be notified of the i
 
 ## Step Three: Create a new branch
```

Comment:
Here we should also encourage checking a relevant branch exists before default creating a new one.

Reply:
How would you suggest we explain this? If a relevant branch exists, we should really be pointing towards the user creating a new branch that stems from the relevant branch. I will try to explain this succinctly in a later revision

Reply:
Yeah I agree runs the risk of information overload in a "quick guide". Maybe we can ignore here on assumption that when using the quick guide you are likely doing a small independent fix and perhaps creating and then deleting the branch ok?

Reply:
I could link to an unlisted page that goes into more depth. That way the information will be in a relevant area, but isn't causing information overload.